packages/flutter/test/material/app_bar_test.dart

```diff
@@ -1514,4 +1514,39 @@ void main() {
 
     expect(getMaterialWidget().shape, roundedRectangleBorder);
   });
+
+  testWidgets('SliverAppBar with shape', (WidgetTester tester) async {
+    const RoundedRectangleBorder roundedRectangleBorder = RoundedRectangleBorder(
+      borderRadius: BorderRadius.all(Radius.circular(15.0)),
+    );
+    await tester.pumpWidget(
+      MaterialApp(
+        home: CustomScrollView(
```

Comment:
Your `const` placement in this block fails `flutter analyze --flutter-repo`. Please make sure analyzer tests are passing.